index.html

```diff
@@ -0,0 +1,126 @@
+        <ul class="menu-bar-ul">
+            <li class="mobile-list"><a href="about.html" class="white">About</a></li>
+            <li class="mobile-list"><a href="#" class="white">Program</a></li>
+            <li class="mobile-list"><a href="#" class="white">Join</a></li>
+            <li class="mobile-list"><a href="#" class="white">Sponsor</a></li>
+            <li class="mobile-list"><a href="#" class="white">News</a></li>
+            <li class="mobile-list"><a href="#" class="navbar-cc-link white">CC Campaign</a></li>
+        </ul>
+    </aside>
+    <nav class="nav">
+        <div class="black-navbar">
+            <ul class="black-navbar-ul">
+                <li><i class= "fa fa-facebook white"></i></li>
+                <li><i class="fa fa-twitter white"></i></li>
+                <li class="white">Logout</li>
+                <li class="white">MyPage</li>
+                <li class="white">Korean</li>
+            </ul>
+        </div>
+        <div class="main-navbar">
+            <ul class="navbar-logo">
+                <li><a href="index.html"><i class="fa fa-life-ring fa-3x orange home-page-link"></i></a></li>
+                <li><h4>CC Global <br> Summit 2015</h4></li>
+            </ul>
+            <i class="fa fa-close fa-2x hamburger white" id="close"></i>
+            <ul class="main-navbar-ul">
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#" class="navbar-cc-link">CC Campaign</a></li>
+            </ul>    
+        </div>
+    </nav>
+    <!--THE MAIN PAGE SECTION-->
+    <section class="first-section">
+        <div class="page-head">
+            <i class="fas fa-bars fa-2x hambuger black"></i>
+            <div><p class="orange hellos">"Hello! Sharing world"</p></div>
+            <h1 class="orange">CREATIVE COMMONS GLOBAL SUMMIT</h1>
+            <p class="orange hello">"Hello! Sharing world"</p>
+        </div>
+        <p class="black intro">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea</p>
+        <h2 class="black date">2015.10.15(THUR) ~ 16(FRI)</h2>
+        <p class="black location">@National Museum of Korea, Art Center Nabi and more</p>
+    </section>
+    <!--THE MAIN PROGRAM SECTION-->
+    <section class="second-section">
+        <h2 class="white">Main Program</h2>
+        <div class="second-section-container">
+            <div class="prog-feature">
+                <img src="./assets/icon-1.png" alt="Lecturer">
+                <h4 class="orange">Lecture</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
```

Comment:
- [ ] The article should be placed similar to the given design. Like we can add some padding and correct the font size.

Original
![image](https://user-images.githubusercontent.com/53019916/127882691-7d6bac44-2485-4053-a453-611fe6675807.png)

Yours
![image](https://user-images.githubusercontent.com/53019916/127882763-ae82bacb-397b-4a1b-8229-9e2c64473d98.png)